Packages/Contrib/cliffordmatrices.html

```diff
@@ -36,11 +36,3 @@ <h3>
         "The Character Table of 2_+^{1+22}.Co_2",
 by Herbert Pahlings, which has been accepted for publication in  J. Algebra.
 </p>
-
-<h3>
-  Contact address
-</h3>
-<p>
-  Herbert Pahlings.
-</p>
```

Comment:
Why? Maybe drop the changes you applied to `Package/Contrib`?

Reply:
We thought that we get rid of the contact data, since there is already an author and maintainer section

Reply:
We removed the contact addresses in that folder as Author/Maintainer are already mentioned. I thought we had agreed on that?

Reply:
Well, point is, I thought the revised plan as of yesterday was that you don't touch `Packages/Contrib` and `Packages/Author` anymore. As it is, these are further changes that add to the already considerable review burden. Please just drop them